.pre-commit-config.yaml

```diff
@@ -13,7 +13,7 @@ repos:
         args:
           - --py39-plus
   - repo: https://github.com/psf/black
```

Comment:
You might want to update this to `https://github.com/psf/black-pre-commit-mirror`, see: https://black.readthedocs.io/en/stable/change_log.html#id9